src/base/cpuinfo.cc

```diff
@@ -58,11 +63,12 @@ static void LoadAvg(CpuLoad &load) {
         load.five_min_avg = averages[1]/num_cpus;
         load.fifteen_min_avg = averages[2]/num_cpus;
     }
-#endif
 }
 
 static void ProcessMemInfo(ProcessMemInfo &info) {
-#ifdef __APPLE__
+#ifdef _WINDOWS
+    //TBD
```

Comment:
Change `TBD` so it's the same text you've used in other PRs please. This would make it far easier to find what needs to be fixed by doing a simple find-in-files "windows-fix`.